doc/source/installation.rst

```diff
@@ -40,7 +40,8 @@ For installing node agent following dependencies have to be manually installed:
 
     $ yum install libffi-devel gcc python-devel openssl-devel
     $ yum install -y http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-1.el7.nux.noarch.rpm
-    $ yum install hwinfo 
+    $ yum install hwinfo
```

Comment:
@brainfunked i am not added hwinfo it comes because of some space change i think